client/packages/common/src/ui/forms/JsonForms/components/Group.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { rankWith, uiTypeIs, LayoutProps, GroupLayout } from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { MaterialLayoutRenderer } from '@jsonforms/material-renderers';
+import { Box, Typography } from '@mui/material';
+
+export const groupTester = rankWith(4, uiTypeIs('Group'));
+
+const UIComponent = (props: LayoutProps) => {
+  const { uischema, schema, visible, renderers, path } = props;
+
+  const layoutProps = {
+    elements: (uischema as GroupLayout).elements,
+    schema: schema,
+    path: path,
+    direction: 'column' as 'column' | 'row',
+    visible: visible,
+    uischema: uischema,
+    renderers: renderers,
+  };
+  return (
+    <Box
+      sx={{
+        maxWidth: 500,
+        paddingLeft: 2,
+        paddingRight: 2,
+        marginBottom: 2,
+      }}
+    >
+      <Typography width="40%" fontSize="1.2em" textAlign="right">
```

Comment:
we may have to use ems throughout 🤔 

Could the font size here come from the theme?

Reply:
Yup, have re-styled the "subtitle" variant in theme with this fontSize -- didn't look like it was being used elsewhere.

Would like to organise the typography a bit better overall at some point: https://github.com/openmsupply/openmsupply-client/issues/1149